internal/validate/kube/eks.go

```diff
@@ -0,0 +1,351 @@
+}
+
+func validateEbsCsiDrivers(testers EbsTestObjects) (result []validate.Result) {
+	result = append(result, validateAddons(testers.addons)...)
+	result = append(result, validateServiceAccount(testers.serviceAccount)...)
+	result = append(result, validateServiceAccountRole(testers.serviceAccountRole)...)
+	result = append(result, validateRolePolicy(testers.ebsRolePolicy)...)
+
+	return result
+}
+
+func validateAddons(addons []string) (result []validate.Result) {
+	for _, addon := range addons {
+		if addon == "aws-ebs-csi-driver" {
+			result = append(result, validate.Result{
+				Status:  validate.Success,
+				Message: "EKS: 'aws-ebs-csi-driver' present in addons",
+			})
+			return result
+		}
+	}
+
+	result = append(result, validate.Result{
+		Status:  validate.Failure,
+		Message: "EKS: no 'aws-ebs-csi-driver' present in addons",
+	})
+
+	return result
+}
+
+func validateServiceAccount(serviceAccount *corev1.ServiceAccount) (result []validate.Result) {
+	if serviceAccount.Name == "ebs-csi-controller-sa" {
+		result = append(result, validate.Result{
+			Status:  validate.Success,
+			Message: "EKS: 'ebs-csi-controller-sa' present on cluster",
+		})
+		return result
+	}
+
+	result = append(result, validate.Result{
+		Status:  validate.Failure,
+		Message: "EKS: no 'ebs-csi-controller-sa' present on cluster",
+	})
+
+	return result
+}
+
+func validateServiceAccountRole(serviceAccountRole string) (result []validate.Result) {
+	if serviceAccountRole != "" {
+		result = append(result, validate.Result{
+			Status:  validate.Success,
+			Message: "EKS: role attached to 'ebs-csi-controller-sa' service account",
+		})
+		return result
+	}
+
+	result = append(result, validate.Result{
+		Status:  validate.Failure,
+		Message: "EKS: no role attached to 'ebs-csi-controller-sa' service account",
+	})
+
+	return result
+}
+
+func validateRolePolicy(rolePolicy RolePolicy) (result []validate.Result) {
+	if *rolePolicy.PolicyName == "AmazonEBSCSIDriverPolicy" {
+		result = append(result, validate.Result{
+			Status:  validate.Success,
+			Message: "EKS: 'AmazonEBSCSIDriverPolicy' bound to role",
+		})
+
+		return result
+	}
+
+	result = append(result, validate.Result{
+		Status:  validate.Failure,
+		Message: "EKS: no 'AmazonEBSCSIDriverPolicy' bound to role",
+	})
+
+	return result
+}
+
+func CurrentContextSetToEKSCluster() error {
+	home := homedir.HomeDir()
+	pathToKubeConfig := filepath.Join(home, ".kube", "config")
+
+	config, err := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
+		&clientcmd.ClientConfigLoadingRules{ExplicitPath: pathToKubeConfig},
+		&clientcmd.ConfigOverrides{
+			CurrentContext: "",
+		}).RawConfig()
+
+	if err != nil {
+		return err
+	}
+
+	got := strings.Split(config.CurrentContext, ":")
+	want := []string{"arn", "aws", "eks"}
+
+	if len(got) >= 3 {
+		got = got[:3]
+		if reflect.DeepEqual(got, want) {
+			return nil
+		}
+	}
+
+	return errors.Newf("%s no eks cluster configured", validate.FailureEmoji)
```

Comment:
```suggestion
return errors.New("no eks cluster configured")
```
Take out emoji here as it's added by the calling function when returning an error